src/gui/accountmanager.h

```diff
@@ -55,14 +56,15 @@ class AccountManager : public QObject {
 
     /**
      * Return a list of all accounts.
-     * (this is a list of QSharedPointer for internal reasons, one should normally not keep a copy of them)
+     * (this is a list of QSharedPointer for internal reasons, one should normally not keep a copy
+     * of them)
```

Comment:
I prefer having to scroll horizontally in that case, but alright I'm fine keeping it at 100.